grass7/imagery/i.sentinel/i.sentinel.download/i.sentinel.download.py

```diff
@@ -108,6 +108,17 @@
 #% guisection: Filter
 #%end
 #%option
+#% key: sleep
+#% description: Sleep time in second before retrying to download data from ESA LTA
+#% guisection: Filter
+#%end
```

Comment:
If not indicated here, it means that this option is not required, correct?

Reply:
Yes, it is useful only if you know that the scenes are offline, but online scenes is not needed